app/api/migrations/migrations/36-populate-mimetype-on-attachments/index.js

```diff
@@ -0,0 +1,31 @@
+import request from 'shared/JSONRequest';
+import { attachmentsPath } from 'api/files/filesystem';
+import mime from 'mime-types';
+
+export default {
+  delta: 36,
+
+  name: 'populate-mimetype-to-attachment',
+
+  description: 'Populates mimetype of an attachment from a url',
+
+  async up(db) {
+    process.stdout.write(`${this.name}...\r\n`);
+    const cursor = await db.collection('files').find({});
+
+    // eslint-disable-next-line no-await-in-loop
+    while (await cursor.hasNext()) {
+      // eslint-disable-next-line no-await-in-loop
+      const file = await cursor.next();
+      if (file.url && !file.mimetype) {
+        // eslint-disable-next-line no-await-in-loop
+        const response = await request.head(file.url);
+        const mimetype = response.headers.get('content-type') || undefined;
```

Comment:
I'm not sure what's the app behavior when there's no mime type. @konzz does it make sense to have a default type like application/octet-stream?

Reply:
Not sure if would be better to leave it empty if we don't know it. We only have a use case now if its image or video, but if in the future we use it for other things, not sure if a default would help or would harm.

Reply:
Ok, let's leave it undefined :+1: thanks